cmd/yorkie/version.go

```diff
@@ -19,26 +19,43 @@ package main
 import (
 	"fmt"
 	"runtime"
+	"runtime/debug"
 
 	"github.com/spf13/cobra"
 
 	"github.com/yorkie-team/yorkie/internal/version"
 )
 
+const commitRevisionKey = "vcs.revision"
+const buildTimeKey = "vcs.time"
+
 func newVersionCmd() *cobra.Command {
 	return &cobra.Command{
 		Use:   "version",
 		Short: "Print the version number of Yorkie",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			fmt.Printf("Yorkie: %s\n", version.Version)
-			fmt.Printf("Commit: %s\n", version.GitCommit)
+			fmt.Printf("Commit: %s\n", getBuildInfoByKey(commitRevisionKey))
```

Comment:
Previously, `GitCommit` is a Short Hash of the revision: `git rev-parse --short HEAD`. How about assigning a short hash here?

Users can still check the full value of the revision using `go version`.
```
$ go version -m ./bin/yorkie
...
	build	vcs.revision=6096695f50be41580f86ea7c017f52d85b400019
	build	vcs.time=2022-08-29T10:42:58Z
	build	vcs.modified=true
```